tasks/celery.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Establish celery task facts
+  set_fact:
+    celery_sysd_path: "/etc/systemd/system/{{ django_stack_celery_svc_name }}.service"
+
+- name: Drop celery config file
+  copy:
+    content: |
+      {% for key in django_stack_celery_svc_conf.keys() %}
+      CELERYD_{{key|upper}}="{{django_stack_celery_svc_conf[key]}}"
+      {% endfor %}
+      CELERYD_CHDIR="{{ active_deploy_dir }}"
+      CELERY_BIN="{{ django_stack_venv_dir }}/bin/celery"
+      CELERY_APP="{{ django_stack_app_name }}"
+    dest: /etc/default/celeryd
```

Comment:
This task should notify a restart/daemon-reload handler, no?

Reply:
Sooo... look below this ... we could add all that logic as a handler if you think it makes more sense. I can see it both ways. The important part is that the systemd daemon SHOULD NOT be daemon-reloaded + restarted before the current process is stopped.

Reply:
oh i see what you are saying though... maybe i still need to add a check for this changing regardless

Reply:
It's complicated: I think the right approach is _yes_ handler but _no_ to register-for-immediate stop.

Reply:
After discussion out of band, added to both. However, the immediate-stop logic isn't a simple append operation, chaining the AND logic: instead, there's a new OR condition that's the registered result of changes to the celery worker config file.